plugins/importexport/medra/classes/MedraWebservice.inc.php

```diff
@@ -57,6 +59,23 @@ function upload($xml) {
 		$arg = "<med:contentID href=\"$attachmentId\" />";
 		return $this->_doRequest('upload', $arg, $attachment);
 	}
+	
+	
+	//
+	// Public Web Service Actions deposit to Crossref
+	//
+	/**
+	 * mEDRA deposit operation.
+	 * @param $xml
```

Comment:
Both of these parameters should be `string`, correct? If so, add the type to the self-documentation, ideally with a description, e.g.:
```
@param $xml string XML document to deposit.
```

Reply:
Done.